### PR TITLE
codecov-cli 9.0.4

### PR DESCRIPTION
**Pull request by dtrodrigues**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Review by daeho-ro — APPROVED** [review]

File: Formula/c/codecov-cli.rb [hunk 1]
```diff
@@ -3,18 +3,18 @@ class CodecovCli < Formula
 
   desc "Codecov's command-line interface"
   homepage "https://cli.codecov.io/"
-  url "https://files.pythonhosted.org/packages/5c/78/c66189c5f0494f2deb84fe734ee5a5d3b388fa6bfe45cca49bfdef32a6f7/codecov-cli-0.8.0.tar.gz"
-  sha256 "d1dc15353a91fc1664745c6ea936c16d22abf03d23d21ebd42c4efe838cb8551"
+  url "https://files.pythonhosted.org/packages/c4/9e/50af9e41e63a74290a447473a7f7f88af931e3e2b7959486bd30e7e16d80/codecov-cli-9.0.4.tar.gz"
```
Comment (chenrui333) [inline Formula/c/codecov-cli.rb:6]: the version seems a bit off?

Comment (daeho-ro) [reply Formula/c/codecov-cli.rb:6]: Wow, I was thought it is 0.9.4

Comment (daeho-ro) [reply Formula/c/codecov-cli.rb:6]: image

I should contact to the upstream, Thanks for the catch @chenrui333 !

Comment (chenrui333) [reply Formula/c/codecov-cli.rb:6]: no worries, I feel like they should forfeit the wrong releases.

**Comment by daeho-ro** [comment]
That's too bad but it seems that they mistakenly publish a release with tag v9.0.1 but it is actually v0.9.1.
Anyway, they confirmed that the version is not wrong, so we may continue to the process, right?

**Comment by chenrui333** [comment]
yeah, totally, I am just thinking of doing things right rather than downgrade later :)

**Comment by chenrui333** [comment]
@thomasrockhu-codecov moving the discussion into the other thread (I certainly think pypi yanked release would address your concern, and help maintain the good shape of the project releases :) )

**Comment by chenrui333** [comment]
for now, I will just close the PR, cc @thomasrockhu-codecov

File: Formula/c/codecov-cli.rb (hunk 1 above)
Comment (dtrodrigues) [reply Formula/c/codecov-cli.rb:6]: Upstream has chosen to go with this version scheme: https://github.com/Homebrew/homebrew-core/pull/197779#issuecomment-2486937465

**Pull request closed without merging**